controllers/interviews.js

```diff
@@ -0,0 +1,71 @@
+const fs = require('fs')
+const Interviews = require('../data/interviews.json')
+
+const listInterviews = (req, res) => {
+    res.status(200).json(Interviews)
+}
+
+const createInterview = (req, res) => {
+    const newInterview = {
+        id: req.query.id,
+        idClient: req.query.idClient,
+        idPostulant: req.query.idPostulant,
+        idApplication: req.query.idApplication,
+        date: req.query.date
+    }
+    Interviews.push(newInterview)
+    fs.writeFile('./data/interviews.json', JSON.stringify(Interviews), {}, (error) => {
+        if (error) {
+            return res.status(400).send(error)
+        }
+        return res.status(201).send(newInterview)
+    })
+}
+
+const updateInterview = (req, res) => {
+    let updatedInterview
+    const updatedInterviews = Interviews.map((interview) => {
+        if (interview.id == req.query.id) {
+            updatedInterview = {
+                id: req.query.id,
+                idClient: req.query.idClient || interview.idClient,
+                idPostulant: req.query.idPostulant || interview.idPostulant,
+                idApplication: req.query.idApplication || interview.idApplication,
+                date: req.query.date || interview.date,
+            }
+            return updatedInterview
+        }
+        return interview
+    })
+    if (!updateInterview) res.status(404).send('Interview NOT found')
+    fs.writeFile('./data/interviews.json', JSON.stringify(updatedInterviews), {}, (error) => {
+        if (error) {
+            return res.status(400).send(error)
+        }
+        return res.status(201).send(updatedInterview)
+    })
+}
+
+const deleteInterview = (req, res) => {
+    let removeInterview
```

Comment:
the variable should be named `removedInterview`. Verbs are not used for this kind of variable.

Anyway you should use deleted instead of removed